code/tyler/django/tyler_django_lab-2/blog_proj/templates/users/register.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %} {% block content %}
```

Comment:
I would love to see more styling on the register/login forms. At least center the form on the page, give it some padding and a link from one form to another. Need an account? Register! Have an account? Log in!

Please apply the Bootstrap classes for styling forms and their submit buttons.